src/entity/plane.rs

```
@@ -51,7 +64,15 @@ impl Plane {
             ),
         }
     }
+
+    pub fn render(&self) {
+        match self.mesh {
+            Some(ref mesh) => mesh.set_position([self.position.x, self.position.y, self.position.z]),
+            None => println!("Can't render w/o graphics!"),
```

Should we raise an Error here instead?

---

panic at the disco!